### PR TITLE
HOTFIX: Fixed issue with updating project details

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Project details could not update because the project identifier was not set currently after I updated the react router.  

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Changes
 * [REST]: Corrected behaviour of date fields in REST API to return epoch instead of textual string. (22.01.1)
 * [Documentation]: Fixed issue that prevented a user from saving analysis results back to a sample and concatenating sequence files. Fixed date format in Swagger REST API documentation. (22.01.2)
 * [REST]: Fixed bug that was preventing syncing remote projects from versions 20.09 and older. (22.01.3)
+* [UI]: Fixed bug preventing updating of project details.
```
Comment (ericenns) [inline CHANGELOG.md:9]: please add (22.01.4) to the end of this changelog entry.

**Review by ericenns — CHANGES_REQUESTED** [review]
Could you please also update the version in the pom file to `22.01.4`

File: CHANGELOG.md (hunk 1 above)
Comment (joshsadam) [reply CHANGELOG.md]: Done

**Review by ericenns — APPROVED** [review]
👍

**Pull request merged**